api/models/event.go

```diff
@@ -4,7 +4,7 @@ import("time")
 
 // TODO: decide which fields are nullable.
 type Event struct {
-	EventID			int			`json:"eventID"`
+	EventID			int			`json:"eventID"`//primary
```

Comment:
Btw, you could add validation for required fields in a struct like on the [`brother.go` model](https://github.com/pacific-theta-tau/tt-db/blob/1670bd93a7eae2d3d869d618c6a042eff74613d8/api/models/brother.go#L6):
```
FirstName   string `json:"firstName" validate:"required"`
```
This will be useful when you implement a POST request for `/api/events`.

For context: this field comes form the [`validator` package](https://github.com/go-playground/validator), and it helps check for all missing fields in a struct in a single code block like [here](https://github.com/pacific-theta-tau/tt-db/blob/1670bd93a7eae2d3d869d618c6a042eff74613d8/api/handlers/brother_handler.go#L140) 
```Go
// snippet from AddBrother() in api/models/brother.go#L6
	// Validate brothers struct
	validate := validator.New()
	if err := validate.Struct(brother); err != nil {
		http.Error(w, err.Error(), http.StatusBadRequest)
		return
	}
```
the `validate.Struct(brother)` returns an error with a list of all missing parameters in the struct `brother`. This could be useful for

Reply:
^ not a change request, just a heads up for you to know that `validator` exists when implementing your other endpoint handlers.